src/libs/Middleware/Logging.js

```diff
@@ -116,12 +105,15 @@ function Logging(response, request) {
                 // Expensify site is down completely OR
                 // Auth (database connection) is down / bedrock has timed out while making a request. We currently can't tell the difference between Auth down and bedrock timing out.
                 Log.hmmm('[Network] API request error: Expensify service interrupted or timed out', logParams);
+            } else if (error.message === CONST.ERROR.THROTTLED) {
+                Log.hmmm('[Network] API request error: Expensify API throttled', logParams);
```

Comment:
Technically it is the user who is throttled? And not the API? Just wondering if this could be a bit more explicit.

Reply:
It's the server, after returning `TOO_MANY_REQUESTS` error.

Reply:
Yeah I see that the server is returning the code. My comment is that the log line is ambiguous. 

- Expensify API throttled a user?
- Expensify API was throttled by something?

Which one is genuinely unclear to me...

Reply:
> * Expensify API throttled a user?

I think it's this. The user's request is definitely throttled by the server. I'm not exactly sure why, but I could update the message to say this. Thoughts?
```
'[Network] API request error: Expensify API throttled the request'
```
cc @johnmlee101 or @madmax330 what are the reasons why a request would be throttled?

Reply:
Maybe `Throttled: User made too many requests` would work?

Reply:
I'm not sure that's accurate, so I would rather be general. Can this be NAB for now?

Reply:
For sure.